fft/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "lambdaworks-fft"
+version = "0.1.0"
+edition = "2021"
```

Comment:
From previous bad experiences, I strongly recommend you set a workspace level version and make these use the workspace one.
See https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table